sovrin_client/test/cli/test_send_node_wrong_node_ip.py

```diff
@@ -0,0 +1,32 @@
+import pytest
```

Comment:
Lets do all the validation for send NODE in one test, no point creating so many tests